remind/cogs/reminders.py

```diff
@@ -542,7 +536,7 @@ def make(value, label):
             labels = 'day hr min sec'.split()
             values = discord_common.time_format(settings.finalcall_before * 60)
             before_str = ' '.join(make(value, label) for label, value in zip(labels, values) if value > 0)
-            desc = f'About to start in {before_str} | {settings.localtimezone}'
+            desc = f'About to start in {before_str}!'
             embed.description = desc
             channel = self.bot.get_channel(settings.finalcall_channel_id)
             msg = await channel.send(role.mention + " " + send_msg, embed=embed)
```

Comment:
Lets add here the name of contest website also
Reasoning being -> Currently when notification comes only role is visible in notification text, we can do something like 
`@Contestant Its Codeforces Time!`. This way when someone sees a notification from the bar, it would be more useful

Reply:
You would need to check for the GLHF! msg alongside